account_banking_pain_base/models/account_payment_order.py

```diff
@@ -24,7 +24,13 @@
 class AccountPaymentOrder(models.Model):
     _inherit = "account.payment.order"
 
-    sepa = fields.Boolean(compute="_compute_sepa", readonly=True, string="SEPA Payment")
+    sepa = fields.Boolean(compute="_compute_sepa", string="SEPA Payment", store=True)
+    sepa_payment_method = fields.Boolean(
+        compute="_compute_sepa",
+        string="SEPA Payment Method",
+        store=True,
```

Comment:
Please don't store these fields that are only for showing a banner, and that will be triggered on module update, making it slower.

Reply:
OK. So I removed the "sepa" field from the account.payment.order tree view, because it is a bad practise to have a non-stored field in a tree view.